### DISCUSSION TITLE
Is the nuxt3 version of i18n SSR enabled?

### DISCUSSION THREAD

**Discussion by urbgimtam** [nuxt-modules/i18n#1814] [Q&A]
Reason why I'm asking is that when page is called via url, it always displays the tag and then quickly switches to the localized string.
Which IMO indicates that the replacement is happening in the client, and not in the server. 

I also thought it might had to do with Vite and being in development, but building it continues the same symptom, 

In Nuxt2, I never experienced these issues, as long as it was in SSR and the component wasn't inside <client-only> tags.

**Comment by ineshbose** [comment]
Reproductions provided after discussing on the Nuxt discord: https://codesandbox.io/p/sandbox/exciting-brook-kwl9nb, https://stackblitz.com/edit/github-us1wuk?file=layouts/default.vue

Solution: missed `vueI18n: { legacy: false }` in i18n config.

**Reply by urbgimtam** [reply]
~~Thank you! That was exactly it.~~

**Update:**
Sorry, I was too fast. It seems on the first time a direct url is called, it does work properly.
However, subsequent direct url calls behave again the same way.

Updated the [codesandbox](https://codesandbox.io/p/sandbox/exciting-brook-kwl9nb,) with `legacy: false`.

**Comment by urbgimtam** [comment]
After searching more deeply, I believe my issue is the same as #1812

**Comment by urbgimtam** [comment]
Seems to be a regression. 

A temporary solution is to manually revert to version 8.0.0-beta.7

[Stackblitz displaying correct funcionality using 8.0.0-beta.7.](https://stackblitz.com/edit/github-us1wuk-fuhiqy?file=package.json)

**Reply by paulvonber** [reply]
downgrade doesn't really help :/

**Reply by CatalinGheorghiu** [reply]
Any news on this one?
